06_predict_survival/nbconverted/survival_mutations_coefs.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Survival mutations: coefficients/predictions analysis
+# 
+# In this notebook, we want to look a bit closer at what features are being selected in the predictions -> survival analysis, to get an idea of which genes are used by the model and what their predictions look like.
+
+# In[1]:
+
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+import mpmp.config as cfg
+import mpmp.utilities.data_utilities as du
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+
+# In[2]:
+
+
+# set results directories
+results_dir = Path(cfg.results_dirs['survival'], 'mutations_me_all')
+
+# set list of PCA component numbers to look for
+n_pcs = 10
+
+
+# In[3]:
+
+
+# order to plot data types in
+training_data_map = {
+    'expression': 'gene expression',
+    'me_27k': '27k methylation',
+    'me_450k': '450k methylation',
+    'vogelstein_mutations': 'all Vogelstein mutations',
+    'significant_mutations': 'significant mutations',
+    'mutation_preds_expression': 'mutation scores, expression',
+    'mutation_preds_me_27k': 'mutation scores, 27k',
+    'mutation_preds_me_450k': 'mutation scores, 450k',
+}
+
+
+# ### Pan-cancer models
+
+# In[4]:
+
+
+def is_feature_covariate(features):
```

Comment:
Can you add a one line function comment to this or some markdown above the table to explain what the is_covariate column means?